syncd/RedisClient.cpp

```diff
@@ -691,6 +691,15 @@ bool RedisClient::hasNoHiddenKeysDefined() const
     return keys.size() == 0;
 }
 
+bool RedisClient::hasNoSwitchDefined() const
```

Comment:
Why there would be no switch? What is purpose of this? You have flag isveryfirstrun